config/samples/dataplane_v1beta1_openstackdataplanenodeset_bgp.yaml

```diff
@@ -26,15 +26,20 @@ spec:
         hostName: edpm-compute-0
         ansible:
           ansibleHost: 192.168.122.100
-          ansibleVars:
-            ctlplane_ip: 192.168.122.100
-            internalapi_ip: 172.17.0.100
-            storage_ip: 172.18.0.100
-            tenant_ip: 172.19.0.100
-            fqdn_internalapi: edpm-compute-0.example.com
   networkAttachments:
     - ctlplane
   nodeTemplate:
+    networks:
+      - name: CtlPlane
```

Comment:
Let's change these network names to lowercase to align with what's in `networkAttachments`.

Reply:
ack, true. Updated them all now